QUIZ.ipynb

```diff
@@ -0,0 +1,76 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "2f3c88f1-18b2-4286-bf48-6c40aa39855c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "11b67077-b3c7-4c54-98cd-7adcc0ecfae0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Hello this is intro file.\n",
+      "My name is om.\n",
+      "My city name is vyara.\n",
+      "My age is 18.\n",
+      "I am create this file for practice.\n",
+      "I practice how to write and read the txt file in python.\n",
+      "I am create this intro file in python and print only even number of row in another file.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "with open('C:\\sqlite3\\intro.txt','r')as f:\n",
+    "    r=f.read()\n",
+    "    print(r)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4ef95b37-8798-4287-ba5d-543de43a15d3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with open('C:\\sqlite3\\intro.txt','r')as f:\n",
```

Comment:
Did you got the even number line then explain me the code....!

Reply:
NO, I don't got the even number.

Reply:
Do you know the logic if you know then please help with that....!

Reply:
NO, I don't know the logic

Reply:
ok bro...!